checks.d/docker_daemon.py

```diff
@@ -96,12 +96,21 @@ def image_tag_extractor(c, key):
     return None
 
 
+def container_name_extractor(c):
+    names = c.get('Names', [])
+    for name in names:
+        # the leading "/" is legit, if there's another one it means the name is actually an alias
+        if name.count('/') <= 1:
+            return [str(name).lstrip('/')]
+    return [c.get('Id')[:11]]
+
+
 TAG_EXTRACTORS = {
```

Comment:
do they though? Reading `image_tag_extractor` it looks like it could return a list of more than one element.